### ISSUE TITLE
Unable to install anything

### ISSUE THREAD

**Issue by pinkjimrussiagirl** [khcrysalis/Feather#38]
i am trying to install 2 .ipa files that i already signed but when i click install it goes to a ready to install popup after packaging and doesn’t do anything after that, not even any apps appeared on home screen

![IMG_7160](https://github.com/user-attachments/assets/59bd7252-4949-42ce-95d9-563ab973de19)
https://github.com/user-attachments/assets/0dc26f93-3750-44fc-abe1-c2822b5b43f7

**Comment by khcrysalis** [comment]
What iOS version are you on?

**Comment by GrPK** [comment]
Having the exact same bug. I'm on 18.1 beta

> What iOS version are you on?

**Comment by khcrysalis** [comment]
Nothing I can do about it, stay off beta software

**Comment by GrPK** [comment]
> Nothing I can do about it, stay off beta software

Can you explain why IOS version affects sideloading? Using other options like Signoulous works as intended.

**Comment by viro2** [comment]
same problem for me on 17.5.1

**Comment by emanuelegreco29** [comment]
Same for me on 17.5.1, 14 Pro Max

**Comment by emanuelegreco29** [comment]
I’m using Feather 1.0.1 with the provided p12 certificate, which still has 278 days available.

On 31 Aug 2024, at 12:17, Bryant ***@***.***> wrote:

﻿

Same for me on 17.5.1, 14 Pro Max

which signing sercive do you use?

**Comment by emanuelegreco29** [comment]
I am currently trying with a leaked certificate from Senumy, should I go for a paid one like KravaSign, MapleSign or such?

On 31 Aug 2024, at 13:12, Bryant ***@***.***> wrote:

﻿

I’m using Feather 1.0.1 with the provided p12 certificate, which still has 278 days available.

I know, which signing service provider do you use? MapleSign? AppTester? Signoulous? Or?

**Comment by MuzahidGithub** [comment]
Please refrain from creating unnecessary mess. it doesn't matter which certificate or signing tool you use. it's a http server and ssl CA hostname related issue. You can't install from a local domain.

**Comment by whoiszr** [comment]
+1 with 13 Pro Max on 17.6.1 (21G101). It was working fine with Feather 1.0, but randomly stopped installing apps. Updated Feather to 1.01 but am still having the same issue.

**Comment by GrPK** [comment]
> +1 with 13 Pro Max on 17.6.1 (21G101). It was working fine with Feather 1.0, but randomly stopped installing apps. Updated Feather to 1.0.1 but am still having the same issue.
> 
> UPDATE: I do use NextDNS and had recently enabled "DNS Rebinding Protection" in the Security tab. This ended up blocking "app.localhost.direct", causing Feather to not install apps. Once I added "app.localhost.direct" to my Allowlist, it is working again.

Yes I did the same as you and now it works even on 18.1. Moreover, I found useful to allowlist these as well:
![image](https://github.com/user-attachments/assets/fa4a7a3f-23c3-490e-a762-d3b7d0489762)

For everyone that have problems, try this, I bet 99% of people having this problem use NextDNS @khcrysalis

**Comment by derduesentrieb** [comment]
> I am not using NextDNS and I am still experiencing this problem, for what it's worth. Using ISP dns

Same here. Installed Feather using Sideloadly and using official Apple dev account. iOS 17.5.1 iPad Pro M4.

**Comment by Askejm** [comment]
Doesn't work for me using default dns or 1.1.1.1 
iOS 18.0.1

**Comment by GottaLoveAng** [comment]
some isps have been blocking the site. ive heard that vpns can fix the issue. online installation mode may be worth a try if you're on 17.7+ as well.

**Comment by Askejm** [comment]
Tried with VPN connected to the US and it doesn't work either. NordVPN

**Comment by GottaLoveAng** [comment]
have you tried online installation mode?

**Comment by dorcso** [comment]
Hi im on ios 26.0.1
Randomly Feather full localhost stopped working and its getting stuck on ready to install i tried reinstalling Feather multiple times but it doesn't work I use dns in the photo I put my allowlist in and my security settings ![image](https://github.com/user-attachments/assets/b7cf52aa-aea7-4d2a-8f60-c2e4274e47b2)
![image](https://github.com/user-attachments/assets/d57dbed3-50da-42fc-bd0b-81faf2bdc4cf)
